src/export/InstanceExporter.ts

```diff
@@ -76,8 +89,9 @@ export class InstanceExporter {
    */
   private setFixedValuesForDirectChildren(
     element: ElementDefinition,
-    existingPath: PathPart[],
-    instanceDef: InstanceDefinition
+    existingPath: string,
+    instanceDef: InstanceDefinition,
+    instanceOfStructureDefinition: StructureDefinition
```

Comment:
Since the function arguments changed, could you also updated the `@param` listing above the function?

Reply:
Ah, I should have been more specific: there's also the new `instanceOfStructureDefinition` that should be added to the `@param` listing.